### PR TITLE
fix(CX-1716): Analytics events triggering multiple times

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-1716]

### Description
- Fix analytics events triggering multiple times

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Fix analytics events triggering multiple times

</details>

[CX-1716]: https://artsyproduct.atlassian.net/browse/CX-1716

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Fix analytics events triggering multiple times)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2659a629a6012d4d4ca44780c79df460fbc74133
</p>

**Comment by MounirDhahri** [comment]
what a mess 😆

File: src/lib/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -370,29 +363,41 @@ export const HomeQueryRenderer: React.FC = () => {
   }, [flash_message])
 
   // Avoid rendering when user is logged out, it will fail anyway
```
Comment (lordkiz) [inline src/lib/Scenes/Home/Home.tsx:365]: @MounirDhahri I think you'd need to remove this comment. It does not describe exactly what comes immediately after.

**Review by lordkiz — APPROVED** [review]
LGTM 🎉

File: src/lib/Scenes/Home/Home.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/lib/Scenes/Home/Home.tsx:365]: oh yes

**Comment by MounirDhahri** [comment]
#squashongreen

**Pull request merged**